pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -100,6 +100,29 @@ Listen Sockets
                                   higher than 0 to enable TCP Fast Open when available.
                                   Default is 0.
 
+.. function:: addDOHLocal(address, certFile(s), keyFile(s) [, url [, options]])
+
+  .. versionadded:: 1.4.0
+
+  Listen on the specified address and TCP port for incoming DNS over HTTPS connections, presenting the specified X.509 certificate.
+
+  :param str address: The IP Address with an optional port to listen on.
+                      The default port is 443.
+  :param str certFile(s): The path to a X.509 certificate file in PEM format, or a list of paths to such files.
+  :param str keyFile(s): The path to the private key file corresponding to the certificate, or a list of paths to such files, whose order should match the certFile(s) ones.
+  :param list url: A list of URLs to accept queries on. The default is /.
```

Comment:
This is a "base" path specifier and not an "exact match". This means that any path under one of the given path will be treated (if one of the URL is "/foo", any DoH request against "/foo/bar?dns=..." will be treated). Maybe we could make that clear here ?

Also accepting a string OR a list of strings would be so nice. I can PR that if needed...